chat-components/src/components/outofofficehourspane/OOOHPane.tsx

```diff
@@ -29,16 +33,14 @@ function OOOHPane(props: IOOOHPaneProps) {
                     role={props.controlProps?.role}
                     className={props.styleProps?.classNames?.containerClassName}
                     dir={props.controlProps?.dir ?? defaultOOOHPaneControlProps.dir}>
-
                     {!props.controlProps?.hideTitle && (decodeComponentString(props.componentOverrides?.title) ||
-                        <Label
+                        <Text
                             className={props.styleProps?.classNames?.titleClassName}
                             styles={titleStyles}
                             tabIndex={-1}
                             id={elementId + "-title"}>
-                            {props.controlProps?.titleText ?? defaultOOOHPaneControlProps.titleText}
-                        </Label>)}
-
+                            <div dangerouslySetInnerHTML={{ __html: displayText }}></div>
```

Comment:
Please test accessibility with this change - whether tis is being announced correctly

Reply:
Verified complete link url is being announced correctly with screen reader